tests/benchdnn/dnn_types.cpp

```diff
@@ -1723,8 +1723,14 @@ void maybe_zero_point(const attr_t &attr, float &d, const int32_t *zero_points,
 }
 
 float compute_eltwise_fwd(pk_t kind, float src, float alpha, float beta) {
-    // don't compute on nan, propagate it
-    if (std::isnan(src)) return NAN;
+    if (std::isnan(src)) {
```

Comment:
is it possible to get here to other edge cases: like `src` being `inf` or `beta` being `NAN`/`inf`?

also, does it make sense to do an early exit (like return 1) if `is_pow0` ..?

Reply:
The reason to handle NaNs in here is the behavior of NaN processing is implementation-defined (when it comes to `input < something` or `input > something`)  while should be pretty specific and benchdnn enforces that. Based on the algorithms supported, this is the first corner case met, since 2019.

If user asked for beta=NaN (which doesn't have any sense from an eltwise operation) they get what they get.

`inf` is handled properly in kernels and standard math functions, no need to handle it.

> does it make sense to do an early exit (like return 1) if is_pow0 ..?

This function just relays the input from benchdnn to the library's CPU implementation with the only exception of NaN handling, such brnch should exist inside a `pow` function and it'll be reached within two extra calls.